environment.yml

```diff
@@ -3,7 +3,7 @@ channels:
   - conda-forge
   - openeye
 dependencies:
-  - python=3.7
-  - jax=0.3.14
-  - jaxlib=0.3.14=cpu*
-  - openeye-toolkits=2020.2.0
+  - python=3.10.10
+  - jax=0.4.7
```

Comment:
nit: any reason not to upgrade jax to latest ([0.4.8](https://github.com/google/jax/releases/tag/jax-v0.4.8))?

Reply:
0.4.8 drops 11.7 CUDA support and I didn't want to also require a cuda update with this change.

Reply:
Oh, but we use CPU-only jax though, right? Does it still somehow require CUDA to be present?

Reply:
Good point, updated to 0.4.8

Reply:
Err, 0.4.8 is not yet in conda, back to 0.4.7

Reply:
Hmm, I don't think jaxlib 0.4.8 has been released yet? https://github.com/google/jax/releases

(IIUC jax and jaxlib do not need to be the same version, as long as jaxlib satisfies jax's lower bound)